.rubocop.yml

```diff
@@ -1,26 +1,18 @@
-require:
-  - rubocop-performance
+inherit_gem:
+  theforeman-rubocop:
+    - strictest.yml
 
 AllCops:
-  NewCops: enable
+  TargetRubyVersion: 2.7
 
 Layout/LineLength:
   Enabled: false
 
 Metrics:
   Enabled: false
 
-Style/Documentation:
```

Comment:
Can we remove all other exceptions and have the config file the same as in the https://github.com/theforeman/hammer-cli-foreman-google/blob/master/.rubocop.yml?

Reply:
@stejskalleos in that case, i'll have to resolve the offenses created by cops like Layout/LineLength, Style/StringLiterals, and Style/FrozenStringLiteralComment. do you want it to be resolved?

Reply:
Yop, that would be great

Reply:
okay, i'll add that changes in this PR, as well as others